ctapipe/io/eventfilereader.py

```diff
@@ -341,6 +342,79 @@ def read(self, allowed_tels=None, requested_event=None,
         return source
 
 
+class DstioFileReader(EventFileReader):
```

Comment:
following the software standards naming conventions, I'd suggest EventIODSTFileReader, or DSTIOFileReader (abbreviations and acronyms should be upper-case)